cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -583,22 +597,22 @@ static Integer concurrentMovements(HttpServletRequest request, boolean isPartiti
   }
 
   static Pattern excludedTopics(HttpServletRequest request) {
-    String excludedTopicsString = request.getParameter(EXCLUDED_TOPICS_PARAM);
-    return excludedTopicsString != null ? Pattern.compile(excludedTopicsString) : null;
+    String parameterString = caseSensitiveParameterName(request, EXCLUDED_TOPICS_PARAM);
+    return parameterString == null ? null : Pattern.compile(request.getParameter(parameterString));
   }
 
   /**
    * @param isUpperBound True if upper bound, false if lower bound.
    */
   static int partitionBoundary(HttpServletRequest request, boolean isUpperBound) {
-    String partitionString = request.getParameter(PARTITION_PARAM);
-    if (partitionString == null) {
+    String parameterString = caseSensitiveParameterName(request, PARTITION_PARAM);
+    if (parameterString == null) {
       return isUpperBound ? Integer.MAX_VALUE : Integer.MIN_VALUE;
-    } else if (!partitionString.contains("-")) {
-      return Integer.parseInt(partitionString);
+    } else if (!parameterString.contains("-")) {
```

Comment:
here should we check `request.getParameter(parameterString)` instead of `parameterString`?